src/git-kill

```diff
@@ -24,15 +28,12 @@ function preconditions {
 
 function steps {
   if [ "$target_branch" = "$initial_branch_name" ]; then
-    if [ "$initial_open_changes" = true ]; then
-      echo "commit_open_changes"
-    fi
+    echo_if_true "commit_open_changes" "$initial_open_changes"
     echo "checkout_main_branch"
   fi
 
-  if [ "$(has_tracking_branch "$target_branch")" = true ]; then
-    echo "delete_remote_branch $target_branch"
-  fi
+  echo_if_true "delete_remote_branch $target_branch" \
```

Comment:
Using `echo_if_true` is getting a bit long and unwieldy here. I'd prefer the previous version, since it is more readable in this particular case.